wgsl/index.bs

```diff
@@ -3745,7 +3745,8 @@ Issue: Which index is used when it's out of bounds?
   <tr algorithm="matrix indexed column vector selection">
        <td class="nowrap">
           |e|: mat|N|x|M|&lt;|T|&gt;<br>
-          |i|: [INT]
+          |i|: [INT]<br>
+          |i| is a `const_expr` expression
```

Comment:
While working through #1792, I realized we don't have a defined term for const_expr and we probably need one. (that's for later.)  This is great for now though.